app/routes/provider.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+import config from 'ember-get-config';
+
+const providers = config.Reviews.providers.slice(1);
+const providerIds = providers.map(p => p.id);
+
+/**
+ * @module ember-osf-reviews
+ * @submodule routes
+ */
+
+/**
+ * @class Provider Route Handler
+ */
+export default Ember.Route.extend({
+    theme: Ember.inject.service(),
+
+    // Todo: Replace the use of hardcoded preprint provider list with an API request.
+
+    beforeModel(transition) {
+        const {slug = ''} = transition.params.provider;
+        const slugLower = slug.toLowerCase();
+        if (providerIds.includes(slugLower)) {
+            if (slugLower !== slug) {
```

Comment:
You should use ember modify the URL here, maybe something [built into route](https://emberjs.com/api/ember/2.14/classes/Ember.Route)?